docs/tutorials/projects/send-security-photo.md

```diff
@@ -0,0 +1,302 @@
+---
+title: "Build a Person Detection Security Robot That Sends You a Photo of the Person Stealing Your Chocolates"
+linkTitle: "Detect a Person and Send a Photo"

```

Comment:
```suggestion
linkTitle: "Build a Person Detection Security Camera"
```
maybe not precisely like this but I think the linkTitle could be more attention grabbing w/ the actual title!